twitter.css

```diff
@@ -0,0 +1,141 @@
+*{  box-sizing: border-box;
+    font-family: sans-serif;
+    
+
+}
+.Container{
+    display: grid;
+    grid-template-columns: 275px auto 350px; /*it can also be written like this instead of writing them out if it has just one column for example no footer and header and other blocks*/
+    outline : 3px solid pink;
+    margin: 0 auto; /*this gives space to the webage at the top . ı.e it places it at the center Also position fixed does not work on grid*/
+    min-height: 650px;
+}
+
+img {
+    width: 50%;
+    height: 50%;
+    object-fit: cover;
+  }
+.home{ 
+   display: grid;
+   Margin: 10px 10px 10px 10px; 
+   height: 150px;
+   outline: 3px solid pink;
+   
+}
+.ticon-logo img {
+    width:30px;
+}
+.ticon-home img {
+    width: 40px;
+}
+.ticon-home {
+    display: grid;
+    grid-auto-flow: column;
+
+}
+   
+
+.left-menu {
+    outline: 3px solid pink;
+    
+}
+
+
+.right-menu {
+    outline: 3px solid pink;
+}
+.right {
+   position: fixed;
+   Margin: 10px 10px 10px 10px;
+   display: grid; 
+}
+.Whats-happening {
+    height:50px;
+    background-color:pink;
+    color: white;
+    outline: 3px solid pink;
+    margin-top: 10px;
+}
+.numbers {
+    display: grid;
+    grid-auto-flow: row;
+    height: 400px;
+    gap: 10px; 
+    margin-top: 5px;
+}
```

Comment:
Separating your grids into various sub-grids is a good way to go! I reckon you can do the same with your tweets as well.